lib/fluent/env_utils.rb

```diff
@@ -0,0 +1,50 @@
+require 'fluent/config'
+
+module Fluent
+  module EnvUtils
+    def self.convert_env_var_to_type(value, type)
```

Comment:
Are this method and `EnvUtils` modules needed?
It seems using `Fluent::Config.reformatted_value` directly is enough.
And moving OjOptions to just under `Fluent` is more simple.